test/utils/specUtils.js

```diff
@@ -8,3 +8,51 @@ function eventFire(el, etype){
     el.dispatchEvent(evObj);
   }
 }
+
+angular.module('patternfly.testutils',['patternfly.utils'])
```

Comment:
I found it a nice way to keep it separated and easy to be pulled in.  I could have added it to the spec file, but they aren't really test specs.